src/main/java/org/sonatype/repository/conan/internal/proxy/ConanMatch.groovy

```diff
@@ -0,0 +1,47 @@
+package org.sonatype.repository.conan.internal.proxy
+
+import org.sonatype.nexus.repository.view.Context
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher.State
+
+/**
+ *
+ * Remote conan servers behave differently to locally running servers in that
```

Comment:
Does the conan project have documentation for this anywhere or is this a bug?

Reply:
The problem is that it is not a specification on how the server is to serve files. So, it is not documented bit also not a bug